addons/account_peppol/models/account_move.py

```diff
@@ -70,3 +71,18 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = False
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False, force_email_company=False, force_email_lang=False):
```

Comment:
Could simplify the override (and ease changes when changing the base method)
```suggestion
    def _notify_by_email_prepare_rendering_context(self, *, model_description=False, **kwargs):
```

Reply:
Heh, runbot doesn't agree with the change, will have a look a it.